### ISSUE TITLE
Windows7 Graphics hardware acceleration 

### ISSUE THREAD

**Issue by dontknowhy** [virtio-win/kvm-guest-drivers-windows#742]
**Is your feature request related to a problem? Please describe.**
I just need a driver to make Windows7 use a hardware acceleration enabled GPU

**Describe the solution you'd like**
I can use VirtIO-vga on Windows7(with hardware accel)
also need qxl-vga hardware accel enabled driver

**Describe alternatives you've considered**
I don't know Win7 will use OpenGL for basic dwm or not?(win8.x will use CPU for dwm progress)

**Additional context**
FILL DXDIAG WITH HARDWARE ACCELERATION
thanks :)

**Comment by YanVugenfirer** [comment]
Unfortunately adding 3D acceleration support for older OSes is completely out of our plans. Sorry.

**Comment by AnaJuliaCodes** [comment]
PLEASE, I WANT

**Comment by ruby-R53** [comment]
yeah same here, windows 7's performance with the QXL device is just dogshit even with the proper and latest drivers installed
virtio-vga already works better than qxl there even without drivers, but because of that it still limits what i can do on my vm
